.github/workflows/container.yml

```diff
@@ -9,7 +9,8 @@ name: Create and publish container
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: ${{ github.repository }}
+  PCW_IMAGE_NAME: ${{ github.repository }}_main
+  K8S_GCE_IMAGE_NAME : ${{ github.repository }}_gce_clean
```

Comment:
I think should be generic for all the providers, as far as I know Dockerfile_k8s will be a generic for all of them